docs/_posts/murat-gunay/2021-12-26-xlm_roberta_large_token_classifier_hrl_xx.md

```diff
@@ -0,0 +1,125 @@
+
+`ORG`, `PER`, `LOC`
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+<button class="button button-orange" disabled>Open in Colab</button>
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/public/models/xlm_roberta_large_token_classifier_hrl_xx_3.3.4_2.4_1640520352673.zip){:.button.button-orange.button-orange-trans.arr.button-icon}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+documentAssembler = DocumentAssembler()\
+      .setInputCol("text")\
+      .setOutputCol("document")
+
+sentenceDetector = SentenceDetectorDLModel.pretrained("sentence_detector_dl", "xx")\
+       .setInputCols(["document"])\
+       .setOutputCol("sentence")
+
+tokenizer = Tokenizer()\
+      .setInputCols(["sentence"])\
+      .setOutputCol("token")
+
+tokenClassifier = XlmRoBertaForTokenClassification.pretrained("xlm_roberta_large_token_classifier_hrl", "xx"))\
+  .setInputCols(["sentence",'token'])\
+  .setOutputCol("ner")
+
+ner_converter = NerConverter()\
+      .setInputCols(["sentence", "token", "ner"])\
+      .setOutputCol("ner_chunk")
+      
+nlpPipeline = Pipeline(stages=[documentAssembler, sentenceDetector, tokenizer, tokenClassifier, ner_converter])
+
+empty_data = spark.createDataFrame([[""]]).toDF("text")
+
+model = nlpPipeline.fit(empty_data)
+text = """يمكنكم مشاهدة أمير منطقة الرياض الأمير فيصل بن بندر بن عبد العزيز في كل مناسبة وافتتاح تتعلق بمشاريع التعليم والصحة وخدمة الطرق والمشاريع الثقافية في منطقة الرياض."""
+result = model.transform(spark.createDataFrame([[text]]).toDF("text"))
+```
+```scala
+val documentAssembler = DocumentAssembler()
+      .setInputCol("text")
+      .setOutputCol("document")
+
+val sentenceDetector = SentenceDetectorDLModel.pretrained("sentence_detector_dl", "xx")
+       .setInputCols(Array("document"))
+       .setOutputCol("sentence")
+
+val tokenizer = Tokenizer()
+      .setInputCols(Array("sentence"))
+      .setOutputCol("token")
+
+val tokenClassifier = XlmRoBertaForTokenClassification.pretrained("xlm_roberta_large_token_classifier_hrl", "xx"))\
+  .setInputCols(Array("sentence","token"))\
+  .setOutputCol("ner")
+
+ner_converter = NerConverter()\
+      .setInputCols(Array("sentence", "token", "ner"))\
+      .setOutputCol("ner_chunk")
+      
+val pipeline = new Pipeline().setStages(Array(documentAssembler, sentenceDetector, tokenizer, tokenClassifier, ner_converter))
+
+val example = Seq.empty["يمكنكم مشاهدة أمير منطقة الرياض الأمير فيصل بن بندر بن عبد العزيز في كل مناسبة وافتتاح تتعلق بمشاريع التعليم والصحة وخدمة الطرق والمشاريع الثقافية في منطقة الرياض."].toDS.toDF("text")
+
+val result = pipeline.fit(example).transform(example)
+```
+</div>
+
+## Results
+
+```bash
++--------------------------+---------+
+|chunk                     |ner_label|
++--------------------------+---------+
+|الرياض                    |PER      |
+|فيصل بن بندر بن عبد العزيز|LOC      |
```

Comment:
This is wrong, that LOC is a Person (Faisal bin Bandar bin Abdulaziz). Also it's bad indented. Probably we need to chose another example and fix indentation.

Reply:
@josejuanmartinez 

- The languages without latin-alphabet like Japanese, Chinese, Arabic, Korean etc have this issue. The indentation may appear like this on the page. I fixed it manually and this is best as far as I can do 😄 
- I took the text from Arabic wiki and I corrected the entities of result section